packages/@ngtools/webpack/src/plugin.ts

```diff
@@ -358,6 +358,14 @@ export class AotPlugin implements Tapable {
       // being built.
```

Comment:
Do we need the changes here?  This is only for Angular 5, isn't it?

Reply:
It's still possible to turn off the new compiler (`--no-experimental-angular-compiler`) so this plugin should still work with Angular 5.